tremor-script/lib/std/size.tremor

```diff
@@ -0,0 +1,7 @@
+fn kibi(size) with

```

Comment:
I'm not sure about the names, while they are technically correct I suspect a lot more people will be used to abbreviations as `mb`, `kb`, or `gb` (which are also technically correct :P) but more helpful to the average person.

Reply:
yes, but the point here is that while maybe for "mebi" and "kibi" it's almost exclusively used for sizes, we can also add "kilo" and "mega", which are used way more widely, and that will lead to an asymmetry

Reply:
They are but they're infrequently used, `mb`, `kb`, `gb`, are frequently used and do not clash with something like `kilo` or `mega` wither. (using uppercase B's also os fine).

but `kibi` and `mebi` arent units that people are working with on a daily basis. `mb`, `kb` are most unix tools report in those units for example.

I don't see a reason why we would want to make this hard for users?

Reply:
Can we at least call them kiB and MiB? those aren't as commonly used, but are out there, and are correct.
I think the issue with units is that there people tend to use incorrect abbreviations, and tough they're technically standardised, nobody in practice cares, and sometimes 1kB means 1024B, sometimes 1000B (and don't even get me started on `KB` and `kb`...).
So the trick here is to find the right balance between correctness and user friendliness

Reply:
I honestly don't know, but I think people will get bitten by it not being the abbreviated that are used on a daily basis (this is a bit of the imperial system problem isn't it?) I don't like it either so. Perhaps it's worth to get some wider feedback from people what they would be expect?

Reply:
With having gotten 5 answers from 4 people on this it seemed like an exercise that didn't move us further :joy: at least we know now that no matter what we choose it's going to be wrong. What you suggested `kiB`/`MiB`/`GB` at least came up twice so let's go with that.